### ISSUE TITLE
Shortened Versions of Most Common Commands?

### ISSUE THREAD

**Issue by kmecpp** [lucko/LuckPerms#397]
Could you please consider adding shorted versions of the most common LuckPerms commands?

Currently to do things as simple as changing the prefix of a group you have to type something like

```
/lp group default meta removeprefix 100
/lp group default meta setprefix 100 &c[&3Member&c]
```

That's a ridiculous amount of stuff I have to remember and type just to change a prefix. Would it not be so much easier to have shortened, more concise, versions of the most common commands? For example, instead of doing the above, what about something like:

`/setprefix default &c[&3Member&c]`

That's a 6 word command to remove the prefix and a 7 word command to reset it, compared to just 3 total. I realize there's no weight in my suggested version, but that could be an optional parameter with a default value of 1 because I think the majority of servers don't even use more than one prefix.

Another example is changing a player's group. This is less tedious but I think a similar thing would still be helpful. Consider the six word:

`/lp user kmecpp parent set default`

Versus three:

`/setgroup kmecpp default`

I understand the usage of every server operator is different, but I think modifying prefixes and setting groups are both universally very common and I'm sure there are other commands as well. Also there are plenty of other ways to implement this `/lp setgroup` or even /lpsg`. Just suggestions :)

**Comment by kmecpp** [comment]
To use tab complete you still have to remember all of the arguments. It's only really helpful if the args are really long (which I'd argue they shouldn't be in the first place). With a bunch of shorter args it just makes it take longer.

**Comment by lucko** [comment]
> That's a ridiculous amount of stuff I have to remember and type just to change a prefix.

No it's not.

> Would it not be so much easier to have shortened, more concise, versions of the most common commands?

How does adding **more** commands and more possible usages make things easier to remember? It does the complete opposite.

Currently, everything follows a logical format.

To edit a user: `/lp user <user>`   
To edit a group: `/lp group <group>`

To edit a permission on a user/group: `/lp user/group <?> permission`   
To edit a parent on a user/group: `/lp user/group <?> parent`   
To edit a prefix/suffix/meta value on a user/group: `/lp user/group <?> meta`   


... and then each of those has it's own subcommand. It is logical and easy to remember once you get started. You can read a command string and understand exactly what it's doing.


If you really "hate typing", you *can* still shorten things down.

For example, `/lp user Luck permission set test.node true` can be shortened to `/lp u Luck p set test.node`.


tl;dr adding more commands does not make things easier to remember.
